Neos.Neos/NodeTypes/Mixin/Site.yaml

```
@@ -0,0 +1,10 @@
+#
+# Abstract NodeType for a Neos.Neos site.
+# Each homepage must extend this NodeType.
+# Nodes of this type must be direct children of the Neos.Neos:Site Root
+# and must not be created at any other place in the tree.
+#
+'Neos.Neos:Site':
+  abstract: true
+  superTypes:
+    'Neos.Neos:Document': true
```

We briefly discussed this in our weekly:
Actually the `uriPathSegment` property should not be part of the homepage.
It would be rather nasty to remove the property from this type even though it is declared in the super type.. But I would say that this is the most pragmatic solution in this case. _If_ the following works:

```suggestion
    'Neos.Neos:Document': true
  properties:
    'uriPathSegments': ~
```

Otherwise we should at least disable the UI editor for this property

---

I do not like it, but I have to admit I don't really see a point were this goes super bad, so lets do it :)

---

I created a separate issue for this: https://github.com/neos/neos-development-collection/issues/4621

as it seems we might not like the unsetting https://github.com/neos/neos-development-collection/pull/4618

---

I agree, thanks!
But maybe we can at least hide it from the inspector by default?